demos/README.md

```diff
@@ -10,17 +10,23 @@
 
 **[Tutorials](../docs/tutorials/README.md)**
 
+**[Useful blogs by ipbabble](https://www.projectatomic.io/blog/author/ipbabble/)**
```

Comment:
I'd remove this useful links section.  Let's just keep it on the top level README.md.  On that file you need to add a link to this README.md